generators/python_functions/html_unescape/config.py

```diff
@@ -0,0 +1,15 @@
+from util.configs import build_generator_function_config
+from util.enums import State
+from . import html_unescape, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_generator_function_config(
+        function=html_unescape,
+        input_example=INPUT_EXAMPLE, 
+        data_type="text",
+        issue_id=233,
+        tabler_icon="IconHtml",
```

Comment:
Could you change this to `BrandHtml5`, please? We are getting the icons from https://tabler-icons-react.vercel.app/. If it's not there, it will run into an error in our CMS. :-)

Reply:
btw, `_template/config.py` still mentions tabler-icons.io. That's where I got the initial icon from ;)

Reply:
Ah yes, have to change that. Thanks. :)